index.html

```diff
@@ -0,0 +1,33 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Learning React</title>
+  <script crossorigin src="https://unpkg.com/react@17/umd/react.development.js"></script>
+  <script crossorigin src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
+  <script src="https://unpkg.com/@babel/standalone/babel.min.js"></script>
+</head>
+<body>
+  <h1>My Website</h1>
+
+  <div id="banner"></div>
+
+  <footer>
+    <p>footer</p>
+  </footer>
+
+  <script type="text/babel">
+    const Banner = () => {
+      return (
+        <div>
+          <p>banner</p>
+        </div>
+      )
+    }
+
+    // render the component to the DOM
+    ReactDOM.render(<Banner />, document.getElementById('banner'))
```

Comment:
Not supported By react 18 should be chnaged to 

`ReactDOM.createRoot(document.getElementById('banner').render(
                    <div>
                         <p>banner</p>
                     </div>
);`